com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/pipeline/ShaderProgramBuilder.java

```diff
@@ -98,13 +101,7 @@ public Task create(IResource input) throws IOException, CompileExceptionError {
         }
 
         compileOptions = modules.getCompileOptions();
-
-        // Include the spir-v flag into the cache key, so we can invalidate the output results accordingly
-        String shaderCacheKey = String.format("output_spirv=%s;output_hlsl=%s;output_wgsl=%s",
-                getOutputSpirvFlag(), getOutputHlslFlag(), getOutputWGSLFlag());
-
```

Comment:
it's now taken into account as part of the builder digest, see `@BuilderParams` -> `paramsForSignature`